src/components/form-elements/_text-input.scss

```diff
@@ -103,12 +104,12 @@ $includeHtml: false !default;
 
     &--valid {
       color: $inputFocusValidBorderColor;
-      font-weight: $fontWeightBold;
+      -webkit-text-stroke: 0.5px $inputFocusValidBorderColor;
```

Comment:
do we need a vendor prefix? I thought sass is adding it by itself?

Reply:
It didn't work without it :/

Reply:
hm, and does it work in firefox/safari/etc.? we will probably also need `-moz` etc.?

Reply:
I checked in both of them and it works